hack/make-rules/test-cmd-util.sh

```diff
@@ -915,6 +915,32 @@ __EOF__
   set +o errexit
 }
 
+# runs specific kubectl create tests
+run_create_tests() {
```

Comment:
since you create a new func here, could we make the name more specific, like `run_dryrun_override_namespace` or sth better ?